packages/cli/src/utils/generate.ts

```diff
@@ -150,17 +151,27 @@ function mergeCMSFile(fileName: string) {
 }
 
 function generateStoreConfigFile(content: any) {
-  return `module.exports = ${JSON.stringify(content, null, 2)}\n`
+  const prettyObject = stringifyObject(content, {
+    indent: '  ',
+    singleQuotes: false,
+  })
+  return `module.exports = ${prettyObject}\n`
 }
 
 async function copyStoreConfig() {
   try {
-    const storeConfigFromStore = await import(userStoreConfigFileDir)
     const storeConfigFromCore = await import(coreStoreConfigFileDir)
+    const storeConfigFromStore = await import(userStoreConfigFileDir)
 
-    const mergedStoreConfig = deepmerge(
-      storeConfigFromCore,
+    // avoid duplicate default values
+    const { default: defaultFromCoreValues, ...otherCoreProps } =
+      storeConfigFromCore
+    const { default: defaultFromStoreValues, ...otherStoreProps } =
```

Comment:
This makes it clear we are discarding the values

```suggestion
    const { default: _, ...otherCoreProps } =
      storeConfigFromCore
    const { default: __, ...otherStoreProps } =
```

Reply:
Nice!